_sections/home-english.md

```diff
@@ -433,6 +429,58 @@ Formulas are used in the [constraint](#constraints), [relevant](#relevant), [cal
 Formulas are composed of functions and operators (+,*,div,etc.). A well-documented full list of operators and functions can be found in the [ODK documentation](https://docs.getodk.org/form-operators-functions/). For the
 technically inclined, the underlying XForms specification is the actual source document for the supported [functions](https://getodk.github.io/xforms-spec/#xpath-functions).
 
+### Look up values in choice lists or attached files
+
+You can look up values from lists defined in the choices sheet, attached CSVs, attached geoJSON files and attached XML files by using the `instance` function. You will use this general structure:
```

Comment:
> general structure (an XPath):

The XPath keyword somewhere in this section may get users pointed in the right direction if they need to search for more help